release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -103,6 +103,9 @@ For more information about enabling a feature set during installation, see xref:
 
 With this release, there are several updates to the *Administrator* perspective of the web console.
 
+* The {product-title} web console displays a `ConsoleNotification` if the cluster is upgrading. Once the upgrade is done, the notification is removed.
+* A *_restart rollout_* option for the deployment resource and a *_retry rollouts_* option for the `DeploymentConfig` custom resource (CR) are available on both the *Action* and *Kebab* menu.
```

Comment:
DeploymentConfig is not an CRD, so it can have CRs

```suggestion
* A *_restart rollout_* option for the Deployment resource and a *_retry rollouts_* option for the `DeploymentConfig` resource are available on both the *Action* and *Kebab* menu.
```

Reply:
Updated. Thank you!